libs/sdk-ui-pivot/src/impl/resizing/tests/updateColDefs.test.ts

```diff
@@ -0,0 +1,99 @@
+// (C) 2007-2021 GoodData Corporation
+
+import {
+    ColumnWidthItem,
+    newAttributeColumnLocator,
+    newWidthForAllColumnsForMeasure,
+    newWidthForAllMeasureColumns,
+    newWidthForAttributeColumn,
+    newWidthForSelectedColumns,
+} from "../../../columnWidths";
+import { TableDescriptor } from "../../structure/tableDescriptor";
+import {
+    testStore,
+    TwoMeasuresWithRowAttribute,
+    TwoMeasuresWithTwoRowAndTwoColumnAttributes,
+} from "./columnSizing.fixture";
+import { MANUALLY_SIZED_MAX_WIDTH, updateColumnDefinitionsWithWidths } from "../agGridColumnSizing";
+import { ReferenceData, ReferenceLdm } from "@gooddata/reference-workspace";
+
+// NOTE: the function under test mutate inputs (by design) so the tests
```

Comment:
Probably missing the end of the sentence

Reply:
fixed